src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Link to SCSS files in HTML is incorrect. SCSS files need to be compiled to CSS and only the resulting CSS file should be linked in the HTML.